master/test/streamdata/stream_trials.sql

```diff
@@ -156,6 +156,115 @@ CREATE TRIGGER stream_trial_project_change_trigger
     AFTER UPDATE OF project_id ON experiments
     FOR EACH ROW EXECUTE PROCEDURE stream_trial_project_change_notify();
 
+-- XXX: more migrations
```

Comment:
I really don't like that this is the way we're doing this, this looks like it will introduce a lot of additional complication if our `add*trigger.sql` files get out of sync with the queries in for testing :/

Could you go ahead and convert addexptrigger.sql to a migration?

Reply:
should I go ahead and do the same for trials and metrics?